tool/ood-gen/bin/watch_scrape.ml

```diff
@@ -52,7 +52,7 @@ let watch_to_yaml t =
         ("category", `String t.category);
       ])
 
-let to_yaml t = `A (List.map watch_to_yaml t)
+let to_yaml t = `O ["watch", `A (List.map watch_to_yaml t)]
```

Comment:
I'm surprised we can change the yaml without needing to change anything else. Was this broken before?

Reply:
I think this broke when yoshi was removed